<think>
No reasoning found</think>
```typescript
    try {
      const _calendar = this.workspace.calendars.byId.get(event.calendar as Ref<ExternalCalendar>)
      if (_calendar !== undefined) {
        await this.remove(event.eventId, _calendar.externalId)
      }
    } catch (err: any) {
      console.error('Remove event error', this.user.workspace, this.user.userId, err.message)
    }
  }

  async syncOurEvents (): Promise<void> {
    this.updateTimer()

```